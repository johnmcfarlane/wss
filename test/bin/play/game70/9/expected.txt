 56  1,9  - HYKE
 53  1,9  - MOKE
 53  1,9  - MOVE
 50  1,9  - HOKE
 50  1,9  - HOVE
 50  1,9  - KOME
 50  1,9  - YOKE
 44  1,9  - HOME
 32  1,7  - YOKEMATE
 28 10,6  | HYEMAL
 28 10,9  - MH
 28 10,9  - MY
 27 10,9  - HM
 26  5,7  | MIKEY
 23  5,7  | MIKE
 23  5,6  | OKIE
 22  1,7  - HYKE
 22  5,7  | KIM
 22 10,9  - ME
 22 10,9  - MO
 21  5,7  | HIKE
 21  5,7  | HIVE
 21  5,7  | KIEV
 21  5,7  | KIHO
 21  1,7  - MOKE
 21  1,7  - MOVE
 21  5,7  | YIKE
 20  1,7  - HOKE
 20  1,7  - HOVE
 20  1,7  - KOME
 20 11,1  | VIDEO
 20  1,7  - YOKE
 19  3,9  - MEO
 19  5,7  | MIEK
 19 11,1  | VIDE
 18 10,9  - HE
 18  5,7  | HIM
 18 10,9  - HO
 18  1,7  - HOME
 18 11,1  | KID
 18  5,7  - MATEY
 18  3,5  - MH
 18  1,5  - OKEH
 17  5,2  | HERMO
 17  2,7  - KYE
 17  2,9  - KYE
 17  2,7  - OKE
 17  2,9  - OKE
 16  4,2  | HOCKY
 16  5,7  | KIE
 16  2,5  - MEH
 16  2,7  - MHO
 16 11,1  | MIDE
 15  2,7  - EMO
 15  2,7  - HYE
 15  2,9  - HYE
 15  5,8  | IKEY
 15 11,1  | MID
 15  4,2  | MOCHY
 15  4,1  | MOECK
 15  2,5  - YEH
 14  5,8  | IVY
 14  5,7  - KATE
 14  5,7  | KI
 14  3,7  - ME
 14  3,9  - ME
 14  3,7  - MO
 14  4,2  | MOCK
 13  4,8  | FYKE
 13  4,5  - HE
 13  4,2  | HECK
 13 11,1  | HIDE
 13  4,2  | HOCK
 13 10,7  | KEMAL
 13  4,2  | KOCH
 13  4,2  | MOCHE
 13  2,7  - VOE
 13  2,9  - VOE
 13  4,2  | YOCK
 12  3,5  - EH
 12  3,7  - HE
 12  3,9  - HE
 12 11,1  | HID
 12  5,7  | HIE
 12  3,7  - HO
 12  5,8  | IKE
 12  5,8  | IVE
 12  8,1  | KMEL
 12  5,7  - MATE
 12  2,7  - MEO
 12  5,2  | MERK
 12  5,2  | MERV
 12  5,7  | MI
 12  5,1  | MOHR
 12  3,5  - OH
 12  3,7  - YE
 12  3,9  - YE
 12  3,7  - YO
 11  4,4  | CHM
 11  9,5  - EMO
 11  5,1  | EMORY
 11  4,9  - EO
 11 10,7  | HEMAL
 11  2,7  - HOE
 11  2,9  - HOE
 11  5,2  | KORY
 11  3,6  - MEOWEd
 11  4,2  | MYC
 11  5,4  | RHO
 11  5,2  | VERY
 11  4,2  | VOCE
 11  4,6  - VOWEd
 11  4,2  | YECH
 11  5,2  | YERK
 11  5,2  | YORK
 10  9,5  - EM
 10  5,7  - HATE
 10  5,2  | HERM
 10  5,7  | HI
 10  4,6  | KEF
 10  4,6  - MEWEd
 10  4,6  - MOWEd
 10 10,8  | MYAL
 10  8,1  | MYEL
 10  4,3  | OCK
 10  4,2  | VOC
 10  5,7  - YATE
 10  5,7  | YI
  9  4,4  | COM
  9  4,3  | ECHO
  9  9,5  - EH
  9  4,6  | EMF
  9 10,8  | EVAL
  9  4,8  | FEM
  9  5,2  | HERY
  9  4,6  - HEWEd
  9  5,2  | HORY
  9  8,1  | KEEL
  9  5,2  | KERO
  9  8,1  | KOEL
  9  5,2  | KORE
  9  4,2  | MOC
  9  5,1  | OKER
  9 10,8  | OVAL
  9  5,1  | OVER
  9 10,8  | VEAL
  9  4,6  - YOWEd
  8  4,4  | CHE
  8  4,4  | COY
  8  3,7  - EO
  8  5,3  | ERK
  8  5,2  | EYR
  8  4,8  | FEH
  8  4,8  | FEY
  8  4,8  | FOH
  8  4,8  | FOY
  8  4,2  | HOC
  8 11,2  | IDEM
  8 10,9  | KAL
  8  5,2  | KER
  8  5,2  | KOR
  8 10,8  | MEAL
  8  5,2  | MERO
  8  5,2  | MORE
  8  4,3  | OCH
  8  3,7  - OE
  8  3,9  - OE
  8  5,1  | OMER
  8 10,9  | VAL
  8  8,2  | VEL
  8  5,2  | VER
  7  4,4  | CH
  7  9,5  - EO
  7 10,8  | HEAL
  7  8,1  | HEEL
  7  5,2  | HERO
  7  5,1  | HOER
  7  5,2  | HORE
  7  5,3  | HR
  7 10,9  | MAL
  7  8,2  | MEL
  7  5,2  | MER
  7  5,2  | MOR
  7  7,9  | OD
  7  7,9  - OI
  7  5,1  | OYER
  7  8,1  | YEEL
  7  5,2  | YORE
  6  4,4  | COE
  6  4,3  | ECO
  6  4,8  | FOE
  6 10,9  | HAL
  6  8,2  | HEL
  6  5,2  | HER
  6  5,2  | HOR
  6  5,3  | ORY
  6  5,6  - OWEd
  6  6,6  - WEdE
  6  5,2  | YER
  6  5,2  | YOR
  5  4,7  | EF
  5  4,8  | FE
  5 11,2  | IDEO
  5  4,7  | OF
  4  8,2  | EEL
  4 11,2  | IDE
  4  5,2  | OER
  4  5,3  | ORE
  4  9,3  | OSE
  3  5,3  | ER
  3  5,8  | IO
  3  5,3  | OR
  2  5,4  | RE
