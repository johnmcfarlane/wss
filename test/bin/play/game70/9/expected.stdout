 53  1,9  - MOKE
 53  1,9  - MOVE
 50  1,9  - HOKE
 50  1,9  - HOVE
 50  1,9  - YOKE
 44  1,9  - HOME
 32  1,7  - YOKEMATE
 23  5,7  | MIKE
 21  5,7  | HIKE
 21  5,7  | HIVE
 21  1,7  - MOKE
 21  1,7  - MOVE
 20  1,7  - HOKE
 20  1,7  - HOVE
 20 11,1  | VIDEO
 20  1,7  - YOKE
 19 11,1  | VIDE
 18  5,7  | HIM
 18  1,7  - HOME
 18 11,1  | KID
 18  5,7  - MATEY
 18 11,1  | VID
 17  2,7  - KYE
 17  2,9  - KYE
 17  2,7  - OKE
 17  2,9  - OKE
 16  2,7  - MHO
 15 11,1  | MID
 14  5,8  | IVY
 14  3,7  - ME
 14  3,9  - ME
 14  3,7  - MO
 14  4,2  | MOCK
 13  4,8  | FYKE
 13  4,2  | HECK
 13 11,1  | HIDE
 13  4,2  | HOCK
 13  2,7  - VOE
 13  2,9  - VOE
 13  4,2  | YOCK
 12  3,7  - HE
 12  3,9  - HE
 12 11,1  | HID
 12  5,7  | HIE
 12  3,7  - HO
 12  5,7  - MATE
 12  5,2  | MERK
 12  5,7  | MI
 12  3,7  - YE
 12  3,9  - YE
 12  3,7  - YO
 11 10,7  | HEMAL
 11  2,7  - HOE
 11  2,9  - HOE
 11  3,6  - MEOWEd
 11  4,2  | MYC
 11  5,4  | RHO
 11  5,2  | VERY
 11  4,6  - VOWEd
 11  4,2  | YECH
 11  5,2  | YERK
 10  9,5  - EM
 10  5,7  - HATE
 10  5,2  | HERM
 10  5,7  | HI
 10  4,6  | KEF
 10  4,6  - MEWEd
 10  4,6  - MOWEd
  9  4,3  | ECHO
  9  9,5  - EH
  9  4,8  | FEM
  9  4,6  - HEWEd
  9  8,1  | KEEL
  9  8,1  | KOEL
  9  5,2  | KORE
  9  4,2  | MOC
  9 10,8  | OVAL
  9  5,1  | OVER
  9 10,8  | VEAL
  9  4,6  - YOWEd
  8  4,4  | COY
  8  4,8  | FEH
  8  4,8  | FEY
  8  4,8  | FOH
  8  4,8  | FOY
  8 11,2  | IDEM
  8  5,2  | KOR
  8 10,8  | MEAL
  8  5,2  | MORE
  8  3,7  - OE
  8  3,9  - OE
  8  5,1  | OMER
  7 10,8  | HEAL
  7  8,1  | HEEL
  7  5,2  | HERO
  7  5,1  | HOER
  7  8,2  | MEL
  7  5,2  | MOR
  7  7,9  | OD
  7  7,9  - OI
  7  5,1  | OYER
  7  5,2  | YORE
  6  4,8  | FOE
  6  5,2  | HER
  6  5,6  - OWEd
  5  4,7  | EF
  5  4,8  | FE
  5  4,7  | OF
  4  8,2  | EEL
  4  5,3  | ORE
  4  9,3  | OSE
  3  5,3  | ER
  3  5,3  | OR
  2  5,4  | RE
